pkg/apis/externaldns/types.go

```diff
@@ -615,7 +615,7 @@ func App(cfg *Config) *kingpin.Application {
 	app.Flag("txt-wildcard-replacement", "When using the TXT registry, a custom string that's used instead of an asterisk for TXT records corresponding to wildcard DNS records (optional)").Default(defaultConfig.TXTWildcardReplacement).StringVar(&cfg.TXTWildcardReplacement)
 	app.Flag("txt-encrypt-enabled", "When using the TXT registry, set if TXT records should be encrypted before stored (default: disabled)").BoolVar(&cfg.TXTEncryptEnabled)
 	app.Flag("txt-encrypt-aes-key", "When using the TXT registry, set TXT record decryption and encryption 32 byte aes key (required when --txt-encrypt=true)").Default(defaultConfig.TXTEncryptAESKey).StringVar(&cfg.TXTEncryptAESKey)
-	app.Flag("txt-new-format-only", "When using the TXT registry, only use new format records which include record type information (e.g., prefix: 'a-'). Reduces number of TXT records (default: disabled)").BoolVar(&cfg.TXTNewFormatOnly)
+	app.Flag("txt-new-format-only", "A key is reserved for TXT migraion. Enabling or disabling it will not take any effect. (deprecated)").BoolVar(&cfg.TXTNewFormatOnly)
```

Comment:
```suggestion
	app.Flag("txt-new-format-only", "Was used during TXT format migration. Enabling or disabling it will not take any effect. (deprecated)").BoolVar(&cfg.TXTNewFormatOnly)
```